tests/spec/unit/podMod.spec.js

```diff
@@ -0,0 +1,107 @@
+var fs = require('fs'),
+	path = require('path'),
+	cordova = require(path.resolve(__dirname, '..', '..', '..', '..', 'cordova-lib/cordova-lib/src/cordova/cordova'));
```

Comment:
Reviewed podModSpec.js. It really should just import podMod.js and test those three functions, independent of anything else (thus unit-tests). Test the work products of the functions to verify since they may not return anything.